superset-frontend/src/explore/components/DatasourcePanel/index.tsx

```diff
@@ -287,6 +344,175 @@ export default function DataSourcePanel({
     [lists.columns, showAllColumns],
   );
 
+  const handleOverwriteDataset = async () => {
+    const { sql, results, dbId } = datasource?.sl_dataset?.query;
+
+    await updateDataset(
+      dbId,
+      datasetToOverwrite.datasetId,
+      sql,
+      // TODO: lyndsiWilliams - Define d
```

Comment:
what is this todo for?

Reply:
I defined d as `any` because I'm not sure what it will be, I wanted to remind myself to define that once data is properly flowing through it and I can figure out what it is.